rules/cloud/aws_s3_data_management_tampering.yml

```diff
@@ -0,0 +1,40 @@
+title: AWS S3 Data Management Tamperin
+id: 78b3756a-7804-4ef7-8555-7b9024a02e2d
+description: Detects when a user tampers with S3 data management in Amazon Web Services.
+author: Austin Songer
+status: experimental
+date: 2021/07/24
+references:
+    - https://github.com/elastic/detection-rules/pull/1145/files
+    - https://docs.aws.amazon.com/AmazonS3/latest/API/API_Operations.html
+    - https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketLogging.html
+    - https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketWebsite.html
+    - https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutBucketEncryption.html
+    - https://docs.aws.amazon.com/AmazonS3/latest/userguide/setting-repl-config-perm-overview.html
+    - https://docs.aws.amazon.com/AmazonS3/latest/API/API_RestoreObject.html
+logsource:
+    service: cloudtrail
+detection:
+    selection_source:
+        - eventSource: iam.amazonaws.com
+    selection_eventname1:
+        - eventName: PutBucketLogging
+    selection_eventname2:
+        - eventName: PutBucketWebsite
+    selection_eventname3:
+        - eventName: PutEncryptionConfiguration
+    selection_eventname4:
+        - eventName: PutLifecycleConfiguration
+    selection_eventname5: 
+        - eventName: PutReplicationConfiguration
+    selection_eventname6: 
+        - eventName: ReplicateObject
+    selection_eventname7: 
+        - eventName: RestoreObject
+    condition: selection_source and selection_eventname1 or selection_eventname2 or selection_eventname3 or selection_eventname4 or selection_eventname5 or selection_eventname6 or selection_eventname7
```

Comment:
why not : 
```
detection:
    selection:
        eventSource: iam.amazonaws.com
        eventName:
           - PutBucketLogging
           - PutBucketWebsite
           - PutEncryptionConfiguration
           - PutLifecycleConfiguration
           - PutReplicationConfiguration
           - ReplicateObject
           - RestoreObject
    condition: selection
```

Reply:
@frack113  wouldn't that mean that each of the events would all have to happened?

Reply:
the logic is `eventSource` AND `eventName( OR every -)` 
When convert with sigmac
before 
`((eventSource:"iam.amazonaws.com" AND eventName:"PutBucketLogging") OR eventName:"PutBucketWebsite" OR eventName:"PutEncryptionConfiguration" OR eventName:"PutLifecycleConfiguration" OR eventName:"PutReplicationConfiguration" OR eventName:"ReplicateObject" OR eventName:"RestoreObject")`
after 
`(eventSource:"iam.amazonaws.com" AND eventName:("PutBucketLogging" OR "PutBucketWebsite" OR "PutEncryptionConfiguration" OR "PutLifecycleConfiguration" OR "PutReplicationConfiguration" OR "ReplicateObject" OR "RestoreObject"))`